neard/src/lib.rs

```diff
@@ -141,6 +142,12 @@ pub fn start_with_config(
         client_actor.clone(),
         view_client.clone(),
     );
+    start_rosettarpc(
```

Comment:
shouldn't this be optional?

Reply:
Absolutely correct. This is not finished yet